lib/curl/CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+cmake_minimum_required (VERSION 2.6)
+project (CGenerator)
+
+cmake_policy(SET CMP0063 NEW)
+
+set(CMAKE_C_VISIBILITY_PRESET default)
+set(CMAKE_CXX_VISIBILITY_PRESET default)
+set(CMAKE_VISIBILITY_INLINES_HIDDEN OFF)
+set(CMAKE_BUILD_TYPE Debug)
+
+set(pkgName "skycoin_rest_api/")
```

Comment:
@stdevMac the slash at the end?
https://github.com/simelo/libskycoin/blob/6d6436361a7a7d30e89a8be5406ac64f3bdb7acb/lib/curl/CMakeLists.txt#L11

Reply:
Done @stdevAlDen